src/auto-evo/RunResults.cs

```diff
@@ -987,6 +988,17 @@ public void LogResultsToTimeline(GameWorld world, List<ExternalEffect>? effects
             }
         }
 
+        public IEnumerator GetEnumerator()
```

Comment:
I'm fine with making this an IEnumerable, but could you add a comment here warning that if auto-evo is running while enumerating this, that's going to randomly cause collection modified exceptions, so don't enumerate this while auto-evo is not ready yet?

Reply:
results is a concurrent dict, it shouldn't?

Reply:
Eh, I use manual locking anyway in this class (which might be unnecessary, but it still exists), so I'd at least warn that this bypasses that locking so using this while auto-evo is running is not suitable.